lib/quadquizaminos_web/views/layout_view.ex

```diff
@@ -49,6 +49,9 @@ defmodule QuadquizaminosWeb.LayoutView do
           <a class = "button" href="<%= Routes.auth_path(QuadquizaminosWeb.Endpoint, :request, "linkedin") %>">
           Sign in with LinkedIn
           </a>
+          <a class = "button" href="<%= Routes.session_path(QuadquizaminosWeb.Endpoint, :new) %>">
```

Comment:
Needs to be the first option in the drop down list

Reply:
worked on it